config/labels/analyzers/clang-tidy.json

```
@@ -1879,7 +1881,9 @@
     ],
     "clang-diagnostic-dangling": [
       "doc_url:https://clang.llvm.org/docs/DiagnosticsReference.html#wdangling",
-      "severity:MEDIUM"
+      "guideline:sei-cert",
+      "sei-cert:MEM50-cpp",
```

```suggestion
      "sei-cert:mem50-cpp",
```

These are lowercase everywhere else.